### PR TITLE
Add name and description to Blazegraph views

### PR DESCRIPTION
**Pull request by olivergrabinski**
#3195

### CODE REVIEW

**Comment by olivergrabinski** [comment]
To review:
1. Both name and description are `String`s.
2. schema.org is not used in the blazegraph json-ld context

File: delta/plugins/blazegraph/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/blazegraph/BlazegraphScopeInitialization.scala [hunk 1]
```diff
@@ -32,6 +32,8 @@ class BlazegraphScopeInitialization(views: BlazegraphViews, serviceAccount: Serv
   implicit private val kamonComponent: KamonMetricComponent = KamonMetricComponent(entityType.value)
 
   private val defaultValue: IndexingBlazegraphViewValue = IndexingBlazegraphViewValue(
+    name = None,
```
Comment (imsdu) [inline delta/plugins/blazegraph/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/blazegraph/BlazegraphScopeInitialization.scala:35]: We need a default value here

File: delta/plugins/blazegraph/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/blazegraph/BlazegraphScopeInitialization.scala [hunk 2]
```diff
@@ -32,6 +32,8 @@ class BlazegraphScopeInitialization(views: BlazegraphViews, serviceAccount: Serv
...
+    description = None,
```
Comment (imsdu) [inline delta/plugins/blazegraph/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/blazegraph/BlazegraphScopeInitialization.scala:36]: Same

**Review by imsdu — CHANGES_REQUESTED** [review]

File: delta/plugins/blazegraph/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/blazegraph/BlazegraphScopeInitialization.scala (hunk 2 above)
Comment (olivergrabinski) [reply delta/plugins/blazegraph/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/blazegraph/BlazegraphScopeInitialization.scala]: Added defaults to `model`

**Review by imsdu — APPROVED** [review]

**Pull request merged**